### ISSUE TITLE
Implement updateChangedAreasOnly

### ISSUE THREAD

**Issue by david-res** [mjs513/ILI9488_t3#23]
Will it be possible to implement the updateChangedAreasOnly function that’s available on the ili9431_t3n and HX8357_t3n libraries?

Thanks,
David

**Comment by mjs513** [comment]
I will take a look to see if it came implement

**Comment by david-res** [comment]
Thanks Mike!

**Comment by KurtE** [comment]
Note: I am looking at ST7735/89 library, and probably NOT going to add it there, 

 

BUT: What I will add is to make it work like ILI9341_t3n (and I believe ILI9488…)

That while ILI9488_t3 code may not have the updateChangedAareasOnly functionality, 

 

I believe it does have the only update to the current clip rectangle.

 

That is if you know you are only changing a specific area of the screen, you can simply set the clip rectangle on that area and do updateScreen

And I think the code is in place to only output for that area.

**Comment by david-res** [comment]
@KurtE thats correct, it does have the clip rectangle functions, I was hoping to try the automatic method that was implemented in the other libraries, but if the performance is the same, then no need to invest time into it. I’ll try on the HX first and see what it performs like
